task/common/machine/storage.go

```diff
@@ -120,6 +124,12 @@ func Transfer(ctx context.Context, source, destination string, include string) e
 		return err
 	}
 
+	if count, size, err := operations.Count(ctx, sourceFileSystem); err == nil {
+		logrus.Infof("Transferring %d files with a total size of %s; this may take a while", count, units.HumanSize(float64(size)))
```

Comment:
```suggestion
		logrus.Infof("Transferring %s (%d files)...", units.HumanSize(float64(size)), count)
```

can we have a "Transferring done" log entry as well?

Reply:
I know that “this may take a while” is a bit too chatty, but do we expect users to be clever enough to correlate file count/size and wait times?

Reply:
> _can we have a "Transferring done" log entry as well?_

https://github.com/iterative/terraform-provider-iterative/pull/463/files#r838792944

Reply:
could [hardcode](https://github.com/iterative/dvc/blob/1a3a2222396301d02f3e076477035e4448725582/dvc/utils/__init__.py#L19) a threshold, e.g. `size > 50 * 2**20 ? "This may take a while." : "Please wait."`

Reply:
I would avoid this level of complication because:
1. Go doesn't have conditional expressions
2. If we hit that code, it's always going to take “a while” in range(2 seconds, infinity)

Reply:
While it's easy to implement, I don't see a clear benefit in the differentiation between “please wait” and “this may take a while”